pyfr/__main__.py

```diff
@@ -118,6 +105,16 @@ def main():
         ap.print_help()
 
 
+def plugin_cli_parser(sp, plugin):
+    cmd = plugin.name
+    for scmd_cfg in plugin.cli:
+        scmd, proc = scmd_cfg[0], getattr(plugin , scmd_cfg[1])
+        ssp = sp.add_parser(scmd, help=f'{cmd} {scmd} --help')
+        ssp.set_defaults(process=proc)
+        for args in scmd_cfg[2:]:
```

Comment:
I'd let the plugin be responsible for configuring this object.  Gives it more control.